eslint-rules/lib/rules/prop-value-shape-deprecation.js

```diff
@@ -130,6 +130,22 @@ module.exports = {
       }
     }
 
+    function getComponentName(node) {
```

Comment:
Why not move to the new `imports` + `getComponentName(componentLocalName, imports)`?

Reply:
I changed to `getComponentLocalName(node)`
It seems like `imports` + `getComponentName` doesn't work here